src/app/utils/appendDimensionsToExtent.js

```diff
@@ -0,0 +1,12 @@
+export const appendDimensionsToExtent = (bib) => {
+    if (!bib.extent || bib.extent.length === 0 || !bib.extent[0]) return bib;
+    
+    let parts = [ bib.extent[0].replace(/\s*;\s*$/, '') ]
+    if (bib.dimensions && bib.dimensions[0]) {
+      parts.push(bib.dimensions[0])
+      parts = parts.join('; ')
+    } else parts = parts[0]
+    bib.extent[0] = parts
```

Comment:
Couldn't this just be
```
    if (bib.dimensions && bib.dimensions[0]) {
      parts.push(bib.dimensions[0])
    }
    
    bib.extent[0] = parts.join('; ')
    ```

Reply:
oh yea i was thinking that would lead to an extra semicolon when there was only one element, but you're right.